styles/layouts/_main-navigation.scss

```diff
@@ -419,10 +419,16 @@
     overflow: hidden;
 
     .dropdown-menu {
+      $navbar-height:70px;
       position: fixed;
-      top: 55px;
+      top: $navbar-height;
       left: 0;
-      width: 98% !important;
+      width: 100%;
+      padding-top:0;
+
+      .dropdown-content {
+        height: 100vh;
```

Comment:
Pls do not break the current UI on PC.

Reply:
it's not breaking. it's under .fixed-class class which is enabled < 1024px